docker-bits/6_remote-desktop.Dockerfile

```
@@ -21,11 +21,11 @@ ENV RESOURCES_PATH="/resources"
 RUN mkdir $RESOURCES_PATH
 
 RUN python3 -m pip install \
-    'git+git://github.com/Ito-Matsuda/jupyter-desktop-server#egg=jupyter-desktop-server'
+    'git+git://github.com/Ito-Matsuda/jupyter-desktop-server#egg=jupyter-desktop-server' 
```

Remove trailing whitespace
```suggestion
    'git+git://github.com/Ito-Matsuda/jupyter-desktop-server#egg=jupyter-desktop-server' 
```